src/govuk/helpers/_links.scss

```diff
@@ -21,9 +21,11 @@
   }
 }
 
-/// Link decoration mixin
+/// Link decoration
 ///
-/// Provides the text decoration for links, including any underline offset
+/// Provides the text decoration for links, including thickness and underline
```

Comment:
Just a general observation: do you need to have both the short summary and the description? It's particularly notable here, where 'Link decoration' on its own doesn't add much. For this and the other mixins, I think you could bring the summary and description together, to avoid duplication - and to avoid the short summary feeling a bit 'floaty'. 

(It's just an idea - I appreciate that this would lead to inconsistency between these mixins and the ones that haven't been updated.)

Reply:
That's a really good idea. Ok if I add it to the list of improvements we plan to make in a future iteration?

Reply:
Yep that's totally fine!